docs/reference/mapping/types/core-types.asciidoc

```diff
@@ -150,7 +150,7 @@ limit to `32766 / 3 = 10922` since UTF-8 characters may occupy at most 3
 bytes.
 
 |`position_offset_gap` |Position increment gap between field instances
-with the same field name. Defaults to 0.
+with the same field name. Defaults to the default of the analyzer which is 10.
```

Comment:
10 seems so arbitrary, but I'm OK with it.  I assume you didn't pick 1 because then a sloppy phrase query with slop=1 would incorrectly match...

But, can we explain why it's non-zero here?  E.g. just add "... so that proximity queries like PhraseQuery do not incorrectly match across 2 values of a multi-valued field"?